### ISSUE TITLE
Save recent searches

### ISSUE THREAD

**Issue by nevyn** [nevyn/spot#2]
Use the empty search screen for listing recent searches and artists and albums recently browsed.

**Comment by Voxar** [comment]
When? Startup only?

**Comment by nevyn** [comment]
yeah, was thinking as the start screen in Search before one made any searches before
